### PR TITLE
Implement Category name translations.

### PR DESCRIPTION
**Pull request by YTG1234**
I love translations.

### CODE REVIEW

File: src/main/java/io/github/hydos/proton/Proton.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public void onInitialize() {
         }
     }
 
-    public static Identifier identifier(String name) {
-        return new Identifier(MOD_ID, name);
+    public static Identifier identifier(String path) {
```
Comment (namebee) [inline src/main/java/io/github/hydos/proton/Proton.java:49]: Path?

**Review by namebee — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/hydos/proton/Proton.java (hunk 1 above)
Comment (YTG1234) [reply src/main/java/io/github/hydos/proton/Proton.java:49]: Yes

Comment (YTG1234) [reply src/main/java/io/github/hydos/proton/Proton.java:49]: That's the name in the `Identifier` constructor

Comment (YTG1234) [reply src/main/java/io/github/hydos/proton/Proton.java:49]: I can revert it

Comment (namebee) [reply src/main/java/io/github/hydos/proton/Proton.java:49]: Revert it.

**Pull request merged**